modules/post-install-edge-aws-extend-cluster.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
+
+:_content-type: CONCEPT
+[id="post-install-edge-aws-extend-cluster_{context}"]
+= Extend existing clusters to use AWS Local Zones
+
+As a post-installation task, you can extend an existing {product-title} cluster on Amazon Web Services (AWS) to use AWS Local Zones.
+
+Extending nodes to Local Zone locations comprises the following steps:
+
+- Adjusting the cluster-network Maximum Transmission Unit (MTU).
```

Comment:
Because these are fragments, they do not need periods. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-periods#lists